### PR TITLE
Jump on Head

### PR DESCRIPTION
**Pull request by tabreturn**
# New Port for Jump on Head

## Game Information
- **Title**: Jump on Head
- **URL**: https://organzola.itch.io/jumponhead

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [ ] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/jumponhead/jumponhead/libs/libzip.so.5
Comment (cdeletre) [inline ports/jumponhead/jumponhead/libs/libzip.so.5:1]: You can add the license https://github.com/cdeletre/PortMaster-MiniDoom2/blob/main/minidoom2/LICENSE.libzip.txt

File: ports/jumponhead/jumponhead/libs/libcrypto.so.1.1
Comment (cdeletre) [inline ports/jumponhead/jumponhead/libs/libcrypto.so.1.1:1]: You can add the license https://github.com/cdeletre/PortMaster-MiniDoom2/blob/main/minidoom2/LICENSE.libcrypto.txt

**Review by cdeletre — CHANGES_REQUESTED** [review]
How about a nice cover ? :)

jumponhead

File: ports/jumponhead/jumponhead/libs/libzip.so.5
Comment (tabreturn) [reply ports/jumponhead/jumponhead/libs/libzip.so.5:1]: Thx. done :)

**Review by cdeletre — APPROVED** [review]

**Review by Cebion — APPROVED** [review]

**Pull request merged**